### ISSUE TITLE
formatting error in post titles

### ISSUE THREAD

**Issue by cpolitano** [Hylozoic/hylo-evo#1141]
Please sanitize formatting for special characters in post titles, such as "&"

<img width="642" alt="Screen Shot 2022-03-14 at 2 29 40 PM" src="https://user-images.githubusercontent.com/5813534/158443139-3023d97a-7032-43e9-be71-8b18395c764d.png">

**Comment by tibetsprague** [comment]
@lorenjohnson is it possible the work you did recently might have impacted/broken this?

**Comment by lorenjohnson** [comment]
Unless something went wrong, this will not happen with posts after this release. A post on March 4th would be before the text formatting release last week. 

The remaining work here is to migrate existing post titles at the database level to decode HTML entities from titles (the only fields that inject HTML into the page are Post.details and Comment.text). Titles and a number of other fields we were over-sanitizing are not HTML fields and shouldn't be being encoded/decoded. Previously both were happening so legacy titles in the DB were encoded and are not now being decoded. We shouldn't cautiously decode as this will cause a regression to a bug Tom fixed a month or so ago such that `<` and `>`, etc will be sanitized out of titles.

NOTE: Only Post.details/description and Comment.text need to be sanitized which always happens on the backend when retrieved.

**Comment by tibetsprague** [comment]
Ah yes, ok I fixed this post just by editing the title and re-saving with an ampersand. Maybe we just run a database query looking for other ampersands to start

**Comment by lorenjohnson** [comment]
Yes, you got it--good instincts. Sanitizing posts before the last release will give us the titles that were presenting before so I think that is the move. I removed "goodfirstticket" as this needs a little care, but is still a small thing.
